codespell_lib/data/dictionary.txt

```diff
@@ -18077,6 +18077,13 @@ scaning->scanning
 scannning->scanning
 scarch->search
 scaricity->scarcity
+scatch->sketch
```

Comment:
Please suggest catch too and equivalents.

Reply:
e0eee9c